forge-gui/res/cardsfolder/upcoming/the_five_doctors.txt

```diff
@@ -0,0 +1,8 @@
+Name:The Five Doctors
+ManaCost:5 G
+Types:Sorcery
+K:Kicker:5
+A:SP$ ChangeZone | Origin$ Library | Destination$ Hand | DestinationAlternative$ Battlefield | ChangeType$ Doctor | ChangeNum$ 5 | ConditionCheckSVar$ X | ConditionSVarCompare$ EQ0 | SubAbility$ DBChangeZone | SpellDescription$ Search your library and/or graveyard for up to five Doctor cards, reveal them, and put them into your hand. If you search your library this way, shuffle. If this spell was kicked, put those cards onto the battlefield instead of putting them into your hand.

```

Comment:
there's no DestinationAlternative + Graveyards missing

Reply:
still not very correct
need pointers?

Reply:
Yes, somehow I didn't see this. What are the remaining issues?

Reply:
- it should normally only be Graveyard, then the OriginAlternative should also include Library
- DestinationAlternative has no place here
- the second effect should just be a mirror of the first (only with inverse condition like you already have + Battlefield instead of Hand)